portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-window.onload = loadPage;
+//window.onload = loadPage;
 
 function loadPage() {
+  getChartAPI();
   getComments();
 }
 
+async function getChartAPI() {
+  var APIScript = document.createElement("script");
+  APIScript.type = "text/javascript";
+  APIScript.src = "https://maps.googleapis.com/maps/api/js?key="+APIkey.APIKey;
+  var chartAPIFile = document.getElementById("APIFile");
+  var head = document.getElementsByTagName("header")[0];
```

Comment:
please consider using `const` instead of `var`, same below
ref https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Thanks!